pkg/golang/build.go

```diff
@@ -147,6 +147,7 @@ func (c Environ) BuildDir(dirPath string, binaryPath string, opts BuildOpts) err
 		"-a", // Force rebuilding of packages.
 		"-o", binaryPath,
 		"-installsuffix", "uroot",
+		"-gcflags=all=-l",
```

Comment:
Please leave a comment in line that says what this is.

Reply:
Added a laconic comment.